admin.go

```diff
@@ -842,3 +853,20 @@ var bufPool = sync.Pool{
 }
 
 var adminServer *http.Server
+
+// adminMetrics - a collection of metrics that can be tracked for the admin API.
+// Call initAdminMetrics to initialize.
+var adminMetrics = struct {
+	requestCount *prometheus.CounterVec
+}{}
+
+func initAdminMetrics() {
+	ns := "admin"
+	sub := "http"
+	adminMetrics.requestCount = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: ns,
+		Subsystem: sub,
+		Name:      "request_count_total",
```

Comment:
This is a bit of "don't repeat yourself". The Prometheus naming convention `_total` implies a counter. It's also good to have the unit be the last part of the metric name so it reads a bit like a sentence. "namespace_subsystem_requests_total" in this case.

```suggestion
		Name:      "requests_total",
```

Reply:
Yeah, I agree - these were lifted from the v1 module (https://github.com/miekg/caddy-prometheus) and I kept them for consistency. But there probably won't be a better time to change them!